.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: go
 go:
-- "1.11"
+- "1.13"
```

Comment:
this doesn't seem right, since we have users of this library that are on Go 1.10 and 1.11. I remember that 1.11.0 had problems but not the later versions like 1.11.9 that we are using in the bidder. Otherwise we might put incompatible code in this repo.